src/tryceratops/settings.py

```diff
@@ -17,6 +21,12 @@
             "formatter": "default",
             "backupCount": 2,
         },
+        "verbose_output": {
+            "class": "logging.StreamHandler",
+            "level": "DEBUG",
+            "formatter": "simple",
+            "stream": "ext://sys.stdout",
+        },
     },
-    "root": {"level": "INFO", "handlers": ["logfile"]},
+    "root": {"level": "INFO", "handlers": ["logfile", "verbose_output"]},
```

Comment:
I don't think it should be here.
If you add it to the root logger, it means **ALL LIBS** will be able to log into our console.

You probably should add a new `loggers` section and add something like "tryceratops" to it.
Are you familiar with how the logging lib works? (If not I can help you by giving more info!)

[Reference](https://docs.python.org/3/library/logging.config.html#object-connections)

Reply:
The ultimate goal would be:

- Any exception from any lib is added to the log file, so we can debug
- When verbosity flag is enabled, only Tryceratops logs are sent to the console

Reply:
I'm not super familiar with the logging library so any pointers would welcome 😀

Reply:
I added a new logger called "tryceratops" and used it instead of the `root` logger and it makes so much more sense now! 😄 Updated PR coming shortly